util/allURDFs.m

```diff
@@ -0,0 +1,22 @@
+function urdfs = allURDFs(rootdir)
```

Comment:
i think we should use this in more of the unit tests.  i found a lot of small bugs this way.